### PR TITLE
Expanded JSON Editor empty state

### PR DESCRIPTION
**Pull request by fiskus**
Screenshot from 2022-09-14 14-45-54

### CODE REVIEW

File: catalog/app/components/JsonEditor/JsonEditor.tsx [hunk 1]
```diff
@@ -3,12 +3,53 @@ import * as R from 'ramda'
...
+
+function EmptyState({ className, noValue, notExpanded }: EmptyStateProps) {
+  if (noValue) {
+    return (
+      <M.Card className={className}>
+        <M.CardContent>
+          <M.Typography variant="h5">JSON editor is empty</M.Typography>
+          <M.Typography variant="body1">
+            Start filling empty rows as in Excel. You can enter values by hand. Type{' '}
+            <Code>{`{}`}</Code> to create objects, or <Code>{`[]`}</Code> to create
```
Comment (nl0) [inline catalog/app/components/JsonEditor/JsonEditor.tsx:29]:
```suggestion
            <Code>{`{}`}</Code> to create an object, or <Code>{`[]`}</Code> to create
```

File: catalog/app/components/JsonEditor/JsonEditor.tsx [hunk 2]
```diff
@@ -3,12 +3,53 @@ import * as R from 'ramda'
...
+            arrays, and then traverse it to enter properties.
```
Comment (nl0) [inline catalog/app/components/JsonEditor/JsonEditor.tsx:30]:
```suggestion
            an array, and then traverse it to enter properties.
```

File: catalog/app/components/JsonEditor/JsonEditor.tsx [hunk 3]
```diff
@@ -3,12 +3,53 @@ import * as R from 'ramda'
 import * as React from 'react'
 import * as M from '@material-ui/core'
 
+import Code from 'components/Code'
 import { EMPTY_SCHEMA, JsonSchema } from 'utils/json-schema'
 
+import illustrationEnterValues from './enter-values.webm'
+import illustrationObjectExpand from './object-expand.webm'
```
Comment (nl0) [inline catalog/app/components/JsonEditor/JsonEditor.tsx:10]: ohwow adding video snippets is something new 👍

File: catalog/app/containers/StoryBook/JsonEditor.tsx [hunk 4]
```diff
@@ -8,16 +8,18 @@ import { JsonValue, ValidationErrors } from 'components/JsonEditor/constants'
...
   },
 }
 const validate = jsonSchema.makeSchemaValidator(schema)
+const noop = () => {}
 
 export default function JsonEditorBook() {
   const [value, setValue] = React.useState<JsonValue>({
-    a: '123',
-    b: 123,
+    a: 123,
```
Comment (nl0) [inline catalog/app/containers/StoryBook/JsonEditor.tsx:20]: is changing a string into a number intentional?

**Review by nl0 — APPROVED** [review]

File: catalog/app/containers/StoryBook/JsonEditor.tsx (hunk 4 above)
Comment (fiskus) [reply catalog/app/containers/StoryBook/JsonEditor.tsx:20]: Yes. To make object invalid

**Comment by fiskus** [comment]
Applied suggested changes, thanks

**Review by nl0 — APPROVED** [review]

**Pull request merged**